explorer/src/lib/components/blocks-card/BlocksCard.svelte

```diff
@@ -29,15 +51,20 @@
   data={blocks}
   {error}
   {loading}
-  className={classes}
   title="Blocks"
-  headerButtonDetails={{ action: () => goto("/blocks"), label: "All Blocks" }}
+  headerButtonDetails={{
+    action: () => loadMoreItems(),
+    disabled: disabledLoading,
+    label: "Show More",
+  }}
 >
   {#if clientWidth > 768}
-    <BlocksTable data={blocks} />
+    <BlocksTable data={displayedBlocks} className="blocks-card__table" />
   {:else}
-    {#each blocks as block (block)}
-      <BlocksList data={block} />
-    {/each}
+    <div class="blocks-card__list">
```

Comment:
Why the extra container?

Reply:
A container is needed for the overflow and height properties to be applied to it.